src/main/java/com/yahoo/identity/services/random/RandomService.java

```diff
@@ -0,0 +1,12 @@
+package com.yahoo.identity.services.random;
+
+import java.security.SecureRandom;
+
+import javax.annotation.Nonnull;
+
+public interface RandomService {
+
+    @Nonnull
+    SecureRandom getRandom();
```

Comment:
`RandomService` should just return the random bytes or returning `Random` so that the underlying implementation can be abstracted.

Reply:
Change to return `byte []`